be/src/http/action/transaction_stream_load.cpp

```diff
@@ -120,6 +120,14 @@ void TransactionManagerAction::handle(HttpRequest* req) {
         return _send_error_reply(req, Status::InvalidArgument(fmt::format("empty label")));
     }
 
+    // HTTP_TRANSACTION_TYPE header is used for multi-statement transactions and should only be sent to FE
+    // (Frontend) nodes. BE (Backend) nodes cannot process multi-statement transaction requests.
+    if (!req->header(HTTP_TRANSACTION_TYPE).empty()) {
+        return _send_error_reply(
+                req, Status::InvalidArgument(
+                             "Multi-statement transaction requests can only be sent to FE nodes, not BE nodes"));
```

Comment:
```suggestion
                             "Multi-statement transaction requests can only be initiated from Frontend."));
```

avoid hard code message BE, may confuse user running shared-data with CN nodes.